hazelcast/src/main/java/com/hazelcast/core/IAtomicLong.java

```diff
@@ -40,6 +40,15 @@
  * </pre>
  * </p>
  *
+ * During a network partition event it is possible for the register to exist in each of the partitioned clusters or to
+ * not exist at all. Under these circumstances the values held in the register may diverge.  Once the network partition
+ * heals, Hazelcast will use the value held in the largest cluster, in this case updates made to the register in the
+ * smaller clusterswill be lost. Starting from Hazelcast 3.10 a merge callback will allow the user to make their own
```

Comment:
`smaller clusterswill be lost` -> `smaller cluster will be lost`

Reply:
`Starting from Hazelcast 3.10...`
I don't see much point in writing what we will (possibly?) have in the future. What if we decide not to have it that way? Or if we don't have time to implement it? I would rather that we simply write what we currently have.

Reply:
:+1: